sql-plugin/src/main/scala/com/nvidia/spark/rapids/basicPhysicalOperators.scala

```diff
@@ -344,6 +353,17 @@ case class GpuProjectAstExec(
     // Process tiers sequentially
     recurse(exprTiers, inputAttrTiers, batch, true)
   }
+
+  def tieredProjectAndClose(cb: ColumnarBatch, opTime: GpuMetric): ColumnarBatch = {
+    val nvtxRange = new NvtxWithMetrics("ProjectExec", NvtxColor.CYAN, opTime)
```

Comment:
nit, we could use `withResource` here for both `nvtxRange` and `cb`

Reply:
Good suggestion.